extensions/spotify-beta/src/helpers/track.ts

```diff
@@ -0,0 +1,26 @@
+export function msToHMS(milliseconds: number): string {
+  const totalSeconds = parseInt(Math.floor(milliseconds / 1000).toString());
+  const totalMinutes = parseInt(Math.floor(totalSeconds / 60).toString());
+  const totalHours = parseInt(Math.floor(totalMinutes / 60).toString());
+  const days = parseInt(Math.floor(totalHours / 24).toString());
+
+  const seconds = parseInt((totalSeconds % 60).toString());
+  const minutes = parseInt((totalMinutes % 60).toString());
+  const hours = parseInt((totalHours % 24).toString());
+
+  const humanized = [pad(hours), pad(minutes), pad(seconds)];
+
+  let time = "";
+  if (days > 0) {
+    time = `${days}:${humanized[0]}:${humanized[1]}:${humanized[2]}`;
+  } else if (hours > 0) {
+    time = `${hours}:${humanized[1]}:${humanized[2]}`;
+  } else if (seconds > 0) {
+    time = `${minutes}:${humanized[2]}`;
+  }
+  return time;
+}
```

Comment:
Not sure if that's worth it but I guess `date-fns` or `day.js` probably has a function doing the same.

Reply:
This is legacy to be honest, do you think we should use data-fns instead?

Reply:
I've been using it in all extensions because the performance cost stays relatively low for greater maintainability. [The package size](https://bundlephobia.com/package/date-fns@2.29.3) has increased lately so maybe it's worth using dayjs instead if possible.